admin-console/src/main/webapp/src/pages/ApplicationPage.tsx

```diff
@@ -158,7 +158,9 @@ export const ApplicationPage: React.FunctionComponent = () => {
                                         onClose={ () => setShowModal(false) }
                                     ><Form isHorizontal>
                                             <FormGroup label='Name' fieldId='name' isRequired
-                                                helperText='This is a short name, only composed of a-z 0-9 and - characters.'>
+                                                helperText={ isEdit ? <HelperText><HelperTextItem variant="warning"  hasIcon>
```

Comment:
Probably the linter is going to complain that there are 2 spaces here
```suggestion
                                                helperText={ isEdit ? <HelperText><HelperTextItem variant="warning" hasIcon>
```